contrib/gitian-keys/README.md

```diff
@@ -1,27 +1,35 @@
 ## PGP keys of Gitian builders and Developers
 
 The file `keys.txt` contains fingerprints of the public keys of Gitian builders
-and active developers.
-
-The associated keys are mainly used to sign git commits or the build results
-of Gitian builds.
-
-The most recent version of each pgp key can be found on most pgp key servers.
+and active developers.  These keys reflect the keys used to sign git commits by
+the developers.  The most recent version of each pgp key can be found on most
+pgp key servers.
 
 Fetch the latest version from the key server to see if any key was revoked in
 the meantime.
-To fetch the latest version of all pgp keys in your gpg homedir,
+
+To fetch the latest version of all pgp keys in your gpg homedir, use the following command:
 
 ```sh
 gpg --refresh-keys
 ```
 
-To fetch keys of Gitian builders and active developers, feed the list of
-fingerprints of the primary keys into gpg:
+To fetch the developer keys, feed the keys.txt file into the following script in order
+to pull the keys from the keyservers:
 
 ```sh
-while read fingerprint keyholder_name; do gpg --keyserver hkp://subset.pool.sks-keyservers.net --recv-keys ${fingerprint}; done < ./keys.txt
+while read fingerprint keyholder_name; do gpg --keyserver hkp://keys.gnupg.net --recv-keys ${fingerprint}; done < ./keys.txt
 ```
 
-Add your key to the list if you provided Gitian signatures for two major or
-minor releases of Bitcoin Core.
+Developers for Veil should make sure they are signing their commits with their public
+key, and ensure the key is what is also loaded into your keybase.io profile.  When your
```

Comment:
Add https:// before keybase.io. Then it will be link.

Reply:
Done.